css/src/components/metricInput.css

```diff
@@ -76,6 +76,7 @@
   padding: 0;
   border: none;
   background: transparent;
+  -webkit-appearance: textfield;
```

Comment:
Please add  `appearance: textfield` in stead, post css will take care of browser vendor style prefix. You can try it here https://autoprefixer.github.io/ 
Please check if `appearance: none` works then use it in stead.